packages/contracts/utils/index.ts

```diff
@@ -14,4 +15,6 @@ export { default as buildArtifacts } from "./buildArtifacts";
 export const expect = chai
   .use(chaiString)
   .use(chaiAsPromised)
-  .use(chaiBignumber(ethers.utils.BigNumber)).expect;
+  .use(chaiBignumber(ethers.utils.BigNumber))
```

Comment:
You shouldn't need this at all -- in fact you can _only_ use `solidity` and remove the imports of the other chai packages. In fact #163 already does this